frontend/gcloud_build.sh

```diff
@@ -0,0 +1,38 @@
+BASE_GAR_DIRECTORY=us-west1-docker.pkg.dev/lifeinthedeep
+IMAGE=${BASE_GAR_DIRECTORY}/ecoviz-explorer/ecoviz-explorer-frontend-staging
```

Comment:
Hardcoding this "staging" service stuff — it seemed more confusing to have to specify "staging" for a deployment to production